website/src/categories/components/combobox/usage.md

```diff
@@ -0,0 +1,157 @@
+---
+title: Combobox usage
+tags: usage
+eleventyNavigation:
+  parent: Combobox
+  key: ComboboxUsage
+---
+
+<section class="no-heading">
+
+<div class="ds-example">
+
+<sl-combobox multiple value='["0","1"]'>
+  <sl-listbox>
+    <sl-option value="0">Mathematics</sl-option>
+    <sl-option value="1">Geography</sl-option>
+    <sl-option value="2">Physics</sl-option>
+    <sl-option value="3">History</sl-option>
+  </sl-listbox>
+</sl-combobox>
+
+</div>
+
+<div class="ds-code">
+
+  ```html
+  <sl-combobox multiple value='["0","1"]'>
+    <sl-listbox>
+      <sl-option value="0">Mathematics</sl-option>
+      <sl-option value="1">Geography</sl-option>
+      <sl-option value="2">Physics</sl-option>
+      <sl-option value="3">History</sl-option>
+    </sl-listbox>
+  </sl-combobox>
+  ```
+
+</div>
+
+</section>
+
+
+<section>
+
+## When to Use
+Use a Combobox to help users easily search, filter, and select options from long list of options.
+
+### **Large Datasets**
+Comboboxes are a better choice than checkbox or radio button groups when dealing with more than 15 options. Unlike static groups that can overwhelm users with lengthy lists, Comboboxes save space and simplify the interface by enabling search and filtering. This makes it easier and faster for users to find their desired option, especially in large datasets like cities, products, or categories, without scrolling through extensive lists.
+
+### Multiple Choices
+Our comboboxes come with multi-selection capabilities provide a clear overview of selected options by displaying them within a tag list. This is ideal when users need to choose multiple items, this ensures selections are visible and easy to manage, allowing users to remove or modify choices effortlessly while keeping the interface organized and intuitive.
+
+</section>
+
+
+<section>
+
+## When Not to Use
+Avoid a Combobox when users benefit more from seeing all options at once for quick selection.
+
+### Fewer Options**
+For list of options small than 15 choices, use [Checkbox](/categories/components/checkbox/usage) or [Radio Button](/categories/components/radio-group/usage) instead of a Combobox. These components present all options visible, making them quicker and more intuitive for users to scan and select without additional interactions.
```

Comment:
```suggestion
For list of options small than 15 choices, use [Checkbox](/categories/components/checkbox/usage) or [Radio Button](/categories/components/radio-group/usage) instead of a Combobox. These components present all options visible without it needing a user interaction first, making them quicker and more intuitive for users to scan and select without additional interactions.
```